packages/packages.go

```diff
@@ -140,6 +142,47 @@ func runWithDeadline(ctx context.Context, timeout time.Duration, cmd string, arg
 	return run(ctxWithTimeout, cmd, args)
 }
 
+func formatFieldsMappingToFormattingString(fieldsMapping map[string]string) string {
+	fieldsDescriptors := make([]string, 0, len(fieldsMapping))
+
+	for name, selector := range fieldsMapping {
+		// format field name and its selector to one single entry separated by ":" and each of them wrapped in quotes
```

Comment:
Nit: can you fix all the comments to start with a capital letter (for consistency with the rest of comments)?

Reply:
Done